### DISCUSSION TITLE
The ability do disarm and hug yourself

### DISCUSSION THREAD

**Discussion by Fogapod** [unitystation/unitystation#7895] [Ideas]
### Description
This could be especially useful for characters with mental disorders. I don't see why you have the ability to punch yourself but not hug or disarm.
I assume there is an if statement that does not allow to apply these to yourself, so the solution would be to just remove it. (also change text to something like `{username} hug themselves` instead of `{username} hugs {username}`.

**Comment by Fogapod** [comment]
I assume these are the lines:
* disarm-  https://github.com/unitystation/unitystation/blob/def4d348a153190c36d0f9a6f8b2b2c67988bfd7/UnityProject/Assets/Scripts/Systems/Inventory/Disarmable.cs#L25
* hug - https://github.com/unitystation/unitystation/blob/f56c51faedb7ef9715e2bcd7298a34851ff51731/UnityProject/Assets/Scripts/Player/Huggable.cs#L17

**Comment by Humonitarian** [comment]
Realistically, you can't exactly disarm yourself, as that is just dropping your weapon. Pushing yourself to the ground I can imagine, but you can roleplay that using "rest".

Hugging yourself also sounds like a nice feature, although it can be roleplayed through *actions already.

**Comment by ilinm1** [comment]
Hugging and disarming yourself looks strange irl and when mood system will be added hugging yourself may lead to abuse

**Reply by Fogapod** [reply]
Looks strange: maybe, but it's possible
May lead to abuse: add a cooldown anywhere
